README.md

```diff
@@ -65,6 +65,25 @@ a one-time manual copy of this dataset to your project location using the [BigQu
 1. Run `bash transfer_dataset.sh <project_id> <dataset> <location>`
 </p></details>
 
+### Hands-on Tutorials
+
+**TODO:** Update cloudshell_git_repo from teamdatatonic to GoogleCloudPlatform X 2 + cloudshell_git_branch to main X 2
```

Comment:
@rahulchowath-dt outstanding TODO?

Reply:
This one will have to done when we merge into the Official Google repo, for this current repo, the parameters are correctly set

Reply:
can you please remove the TODO and create a separate task in the project tracker?

Reply:
[Task](https://www.notion.so/datatonic/Cloud-Shell-Tutorial-Repo-Branch-Repointing-ea2d8c093e504c2cab8da9213baab2ab) added in the Project tracker